Makefile

```diff
@@ -19,7 +19,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-SHELL := /bin/bash
+SHELL ?= /bin/bash
```

Comment:
support override

Reply:
Its not clear why we need an override, but regardless
1. Please make this in a separate PR
2. These files are synced from https://github.com/istio/common-files/tree/master/files, so they will need to be changed there to have effect (same with the run.sh change)

Reply:
I'm on nixos and paths are different. I'll just back out these changes for now and open up new PR if needed